pom.xml

```diff
@@ -3,8 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <!--JENKINS-18537-->
-    <version>1.554.1</version>
+    <version>1.580.1</version>
```

Comment:
@KostyaSha 

> This doesn't mean that you must update core dependencies in non workflow plugins

`SCMTriggerItem` was added in 1.568. 1.580.x is the next LTS after that.